src/SwaggerProvider.DesignTime/v2/Parser/SchemaParserExceptions.fs

```diff
@@ -6,30 +6,26 @@ type SwaggerSchemaParseException(message) =
     inherit Exception(message)
 
 /// Schema object does not contain the `field` that is Required in Swagger specification.
-type FieldNotFoundException<'T>(obj:'T, field:string, specLink:string) =
-    inherit SwaggerSchemaParseException(
-        sprintf "Object MUST contain field `%s` (See %s for more details).\nObject:%A"
-            field specLink obj)
+type FieldNotFoundException<'T>(obj: 'T, field: string, specLink: string) =
+    inherit SwaggerSchemaParseException(sprintf "Object MUST contain field `%s` (See %s for more details).\nObject:%A" field specLink obj)
 
 /// The `field` value is not specified in Swagger specification
-type UnknownFieldValueException<'T>(obj:'T, value:string, field:string, specLink:string) =
-    inherit SwaggerSchemaParseException(
-        sprintf "Value `%s` is not allowed for field `%s`(See %s for more details).\nObject:%A"
-            value field specLink obj)
+type UnknownFieldValueException<'T>(obj: 'T, value: string, field: string, specLink: string) =
+    inherit SwaggerSchemaParseException(sprintf
+                                            "Value `%s` is not allowed for field `%s`(See %s for more details).\nObject:%A"
+                                            value
+                                            field
+                                            specLink
+                                            obj)
```

Comment:
@nojaf @yisusalanpng here I also a little bit surprised
```fsharp
    inherit SwaggerSchemaParseException(sprintf
                                             "Value `%s` is not allowed for field `%s`(See %s for more details).\nObject:%A"
                                             value
                                             field
                                             specLink
                                             obj)
```

why Fantoms does not insert line break before `sprintf`?